### DISCUSSION TITLE
Prometheus and Alertmanager on armv8

### DISCUSSION THREAD

**Discussion by korontanis** [carlosedp/cluster-monitoring#163] [Q&A]
Hi,

I just wanted to ask if the Docker images of Prometheus and Alertmanager that you provide are supporting armv8. I have used the monitoring stack on armv7 several times and it works perfectly. Today I tried to deploy it on armv8, but Prometheus pod and Alert manager pod were not generated. Is there something that I can do? If there are no Docker images of armv8, could you possibly build some?

Thanks

**Comment by carlosedp** [comment]
The images used by the monitoring stack come from official Prometheus builds at https://hub.docker.com/r/prom/prometheus.

Maybe they started buiding for it recently. Try to bump the version on `image_sources_versions.jsonnet` and regenerate the manifests.
